### PR TITLE
fix log.meta will be truncate when cdc restart using s3 sink

### PR DESCRIPTION
**Pull request by yuqi1129**
### What problem does this PR solve? 
see https://github.com/pingcap/ticdc/issues/1400

### Check List 

Tests 

 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update key monitor metrics in both TiCDC document and official document

### CODE REVIEW

**Comment by liuzix** [comment]
Hi, thank you for contributing to TiCDC. If you need any help or information, feel free to discuss here or on the PingCAP community slack channel.

@5kbpers Could you keep an eye on this PR? Thanks!

**Comment by yuqi1129** [comment]
Can anyone help to view this PR ?

**Comment by dengqee** [comment]
@3pointer PTAL

**Comment by yuqi1129** [comment]
@3pointer Hi, can you take time to view this PR?

File: cdc/owner.go [hunk 1]
```diff
@@ -367,13 +367,17 @@ func (o *Owner) newChangeFeed(
 		sinkTableInfo[j-1] = new(model.SimpleTableInfo)
 		sinkTableInfo[j-1].TableID = tid
 		sinkTableInfo[j-1].ColumnInfo = make([]*model.ColumnInfo, len(tblInfo.Cols()))
+		// forget to assign value to schema and table, or file log.meta about this content are all empty
+		sinkTableInfo[j-1].Schema = table.Schema
```
Comment (yuqi1129) [inline cdc/owner.go:371]: @3pointer , It seems that this point has been fixed see #1672 , should this PR need closed?

Comment (ben1009) [reply cdc/owner.go:371]: > @3pointer , It seems that this point has been fixed see #1672 , should this PR need closed?

@3pointer could u help confirm, thanks

Comment (amyangfei) [reply cdc/owner.go:371]: Yes, changed here can be reverted.

**Comment by 3pointer** [comment]
@yuqi1129 Yes it is resolved by #1463. Sorry we are unaware of there are similar issues and this PR first.

**Comment by yuqi1129** [comment]
@3pointer, That's right,  as the related issues have been fixed, i will close this PR later

**Comment by hi-rustin** [comment]
> @3pointer, That's right, as the related issues have been fixed, i will close this PR later

Thank you for your contribution! Again, we apologize for not noticing your changes first! We are very sorry! 💚 💙 💜 💛 ❤️

**Comment by hi-rustin** [comment]
/close 

Close it because the related issue has been fixed.

**Pull request closed without merging**